.circleci/config.yml

```diff
@@ -55,7 +55,19 @@ references:
     branches:
       ignore: /.*/
 
-version: 2
+orbs:
+  slack: circleci/slack@3.4.2
+
+version: 2.1
+
+commands:
+  send_nightly_build_failed_notification:
+    steps:
+      - slack/status:
+          fail_only: true
+          failure_message: Nightly Build failed. Please check the cause to avoid further issues.
+          include_job_number_field: false
+          webhook: ${TEAM_SLACK_WEBHOOK}
```

Comment:
This webhook key name could be different
- if you set the webhook in the **Circle's Project Settings**, `SLACK_WEBHOOK` is the key name
- if you set the webhook in the **Vault**, `TEAM_SLACK_WEBHOOK`  is the key name

Reply:
⚠️ in the n-health vault, ops cops team channel webhook was set as `TEAM_SLACK_WEBHOOK`.